portal/views/user.py

```diff
@@ -957,6 +957,89 @@ def delete_user_consents(user_id):
     return jsonify(message="ok")
 
 
+@user_api.route('/user/<int:user_id>/encounter', methods=('GET',))
```

Comment:
@pbugni Paul, is the `user_id` required if only current user is expected?  I think that the frontend will need to get the user id (e.g. maybe call to /api/me) before it calls this api endpoint, just thinking that it probably doesn't need to make that extra call to get the user id.